ontology-service/src/test/java/no/sikt/nva/OntologyControllerTest.java

```diff
@@ -36,11 +47,72 @@ public static void cleanupSpec() {
     }
 
     @Test
-    void shouldReturnOk() {
-        AwsProxyRequest request = new AwsProxyRequestBuilder("/file", HttpMethod.GET.toString())
-                .header(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON)
+    void shouldReturnTurtle() {
+        AwsProxyRequest request = new AwsProxyRequestBuilder("/test", HttpMethod.GET.toString())
+                .header(HttpHeaders.ACCEPT, "application/turtle")
+                .build();
+        AwsProxyResponse response = handler.handleRequest(request, lambdaContext);
+        assertEquals(HttpStatus.OK.getCode(), response.getStatusCode());
+        assertEquals("_:b0 a \"test\" .", response.getBody());
+    }
+
+    @Test
+    void shouldReturnNTriples() {
+        AwsProxyRequest request = new AwsProxyRequestBuilder("/test", HttpMethod.GET.toString())
+                .header(HttpHeaders.ACCEPT, "application/ntriples")
                 .build();
         AwsProxyResponse response = handler.handleRequest(request, lambdaContext);
-        Assertions.assertEquals(HttpStatus.OK.getCode(), response.getStatusCode());
+        assertEquals(HttpStatus.OK.getCode(), response.getStatusCode());
+        assertEquals("_:b0 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> \"test\" .",
+                response.getBody());
+    }
+
+    @Test
+    void shouldReturnNotFound() {
+        AwsProxyRequest request = new AwsProxyRequestBuilder("/not-found", HttpMethod.GET.toString())
+                .header(HttpHeaders.ACCEPT, "application/ntriples")
+                .build();
+        AwsProxyResponse response = handler.handleRequest(request, lambdaContext);
+        assertEquals(HttpStatus.NOT_FOUND.getCode(), response.getStatusCode());
+    }
+
+    @Test
+    void shouldReturnRdfXml() {
+        AwsProxyRequest request = new AwsProxyRequestBuilder("/test", HttpMethod.GET.toString())
+                .header(HttpHeaders.ACCEPT, "application/rdf+xml")
+                .build();
+        AwsProxyResponse response = handler.handleRequest(request, lambdaContext);
+        assertEquals(HttpStatus.OK.getCode(), response.getStatusCode());
+        assertEquals("<rdf:RDF xmlns:rdf=\"http://www.w3.org/1999/02/22-rdf-syntax-ns#\" />",
+                response.getBody());
+    }
+
+    @Test
+    void shouldReturnJsonLd() {
+        AwsProxyRequest request = new AwsProxyRequestBuilder("/test", HttpMethod.GET.toString())
+                .header(HttpHeaders.ACCEPT, "application/ld+json")
+                .build();
+        AwsProxyResponse response = handler.handleRequest(request, lambdaContext);
+        assertEquals(HttpStatus.OK.getCode(), response.getStatusCode());
+        assertEquals("{\"@context\": {\"@vocab\": \"https://example.org/\"}, \"name\": \"test\"}",
```

Comment:
Is this the same content as the file? Why not read the file here? You can import nva-commons:core in testImplementation to read from resources or you re-use the method `getResourceAsString`